inst/extdata/qualification-workflow-template.R

```diff
@@ -61,6 +77,10 @@ createQualificationReport <- function(qualificationRunnerFolder,
   #' If not set, report created will be named `report.md` and located in  `reOutputFolder`
   reportName <- file.path(reOutputFolder, "report.md")
   
+  #' Provide a template docx document passed to Pandoc for the conversion of the md report into docx
+  #' Default template is available in `system.file("extdata", "reference.docx", package = "ospsuite.reportingengine")`
+  wordConversionTemplate <- NULL
```

Comment:
Let's provide wordConversionTemplate as an argument of createQualificationReport